subjects/localhost/README.md

```diff
@@ -0,0 +1,162 @@
+## Localhost
+
+The Hypertext Transfer Protocol (HTTP) is an application layer protocol in the Internet protocol suite model for distributed, collaborative, hypermedia information systems.
+HTTP is the foundation of data communication for the World Wide Web. Hypertext documents include hyperlinks to other resources that the user can easily access, for example, by a mouse click or tap on the screen in a web browser.
```

Comment:
Those two paragraphs are the same of other schools that created similar projects.
I suggest to review completely the introduction or, in the worst case scenario, to quote explicitly the source (which is wikipedia)